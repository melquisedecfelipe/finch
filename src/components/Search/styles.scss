.search-form {
  display: flex;
  align-items: center;
  height: 35px;
  background: #FFFFFF;
  box-shadow: 0px 3px 6px #2959B729;
  border-radius: 90px;
  border: 0;
  margin: 0;
  padding: 0;

  > img {
    width: 17px;
    margin: 10px;
  }

  > input {
    width: 250px;
    height: 20px;
    border: 0;
    font-size: 14px;
    padding: 0 10px;
    color: #7D7D7D;
    border-top-right-radius: 90px;
    border-bottom-right-radius: 90px;
    border-left: 1px solid #CCCCCC;
  }
}

@media screen and (max-width: 768px) {
  .search-form {
    width: 100%;
  }
}
