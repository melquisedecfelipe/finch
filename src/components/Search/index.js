/* eslint-disable react/prop-types */
import React from 'react';

import './styles.scss';

import Lupe from '../../assets/loupe@2x.png';

export default function Search(props) {
  const { handleInput } = props;

  return (
    <div className="search-form">
      <img src={Lupe} alt="Lupa" />
      <input type="text" placeholder="Buscar" onChange={handleInput} />
    </div>
  );
}
