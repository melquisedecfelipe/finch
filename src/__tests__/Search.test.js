import React from 'react';
import { shallow } from 'enzyme';
import Search from '../components/Search';

describe('Search page testing with Enzyme', () => {
  it('renders without crashing', () => {
    const wrapper = shallow(<Search />);

    expect(wrapper).toMatchSnapshot();
  });
});
