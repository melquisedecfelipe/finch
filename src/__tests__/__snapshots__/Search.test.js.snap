// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search page testing with Enzyme renders without crashing 1`] = `ShallowWrapper {}`;
